quick_tour/the_big_picture.rst

```diff
@@ -1,267 +1,182 @@
 The Big Picture
 ===============
 
-Start using Symfony in 10 minutes! This chapter will walk you through the
-most important concepts behind Symfony and explain how you can get started
-quickly by showing you a simple project in action.
+Start using Symfony in 10 minutes! Really! That's all you need to understand the
+most important concepts and start building a real project!
 
 If you've used a web framework before, you should feel right at home with
-Symfony. If not, welcome to a whole new way of developing web applications.
+Symfony. If not, welcome to a whole new way of developing web applications. Symfony
+*embraces* best practices, keeps backwards compatbility (Yes! Upgrading is always
+safe & easy!) and offers long-term support.
 
 .. _installing-symfony2:
 
-Installing Symfony
-------------------
+Downloading Symfony
+-------------------
 
-Before continuing reading this chapter, make sure to have installed both PHP
-and Symfony as explained in the :doc:`/setup` article.
+First, make sure you've installed `Composer`_ and have PHP 7.1.3 or higher.
 
-Understanding the Fundamentals
-------------------------------
+Ready? In a terminal, run:
 
-One of the main goals of a framework is to keep your code organized and
-to allow your application to evolve easily over time by avoiding the mixing
-of database calls, HTML tags and other PHP code in the same script. To achieve
-this goal with Symfony, you'll first need to learn a few fundamental concepts.
+.. code-block:: terminal
 
-When developing a Symfony application, your responsibility as a developer
-is to write the code that maps the user's *request* (e.g. ``http://localhost:8000/``)
-to the *resource* associated with it (the ``Homepage`` HTML page).
+    $ composer create-project symfony/skeleton quick_tour
 
-The code to execute is defined as methods of PHP classes. The methods are
-called **actions** and the classes **controllers**, but in practice most
-developers use **controllers** to refer to both of them. The mapping between
-user's requests and that code is defined via the **routing** configuration.
-And the contents displayed in the browser are usually rendered using
-**templates**.
+This creates a new ``quick_tour/`` directory with, at first, just *one* file:
```

Comment:
I find this explanation about having just 1 file called `composer.json` confusing. Users won't ever see this single file but the entire project files, so I'd recommend to remove this.